docs/usage.rst

```diff
@@ -4,32 +4,40 @@
 Usage
 =====
 
-Create a :class:`~parsel.selector.Selector` object for the HTML or XML text
-that you want to parse::
+Create a :class:`~parsel.selector.Selector` object for the input text that you want to parse and use.
```

Comment:
Please, keep the original line length limit of 79.

Also, not a strong opinion, but I don’t see why you felt the need to add “and use” at the end. I would rather simplify the line further instead:

```suggestion
Create a :class:`~parsel.selector.Selector` object for your input text.
```

Reply:
Those are from the original usage, I didnt put them there.